### ISSUE TITLE
Updated ansible version parsing fails with releases not from git repo

### ISSUE THREAD

**Issue by rootwork** [geerlingguy/drupal-vm#1736]
### Issue Type

 - Bug Report / Support Request

### Your Environment

```
Vagrant 2.0.3
VirtualBox 5.2.8r121009
ansible 2.4.3.0
  config file = None
  configured module search path = [u'/Users/foo/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/2.4.3.0_4/libexec/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.14 (default, Mar 10 2018, 00:01:04) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
```

### Your OS

 - **macOS** (10.12.6)

### Summary

#1726 changed the way the ansible version is parsed to allow for git repo releases, but it now includes extraneous strings when ansible is installed via pip or homebrew, then fails. You'll notice in "your environment" above that `ansible --version` returns not only the version but some other settings. That entire string is then passed into the version parsing, which fails. You can verify it here, using the regex from that change:

http://rubular.com/r/aFs8eiAxV7

A similar string is returned when ansible is installed via pip.

Prior to #1726, the version number was successfully parsed.

**Comment by geerlingguy** [comment]
Gah! It will never be working 100% I guess :(

**Comment by oxyc** [comment]
https://github.com/geerlingguy/drupal-vm/issues/1725#issuecomment-374705142

**Comment by geerlingguy** [comment]
Fixed in PR #1740.
